media/main-bundle.js

```diff
@@ -486,6 +600,16 @@ function petPanelApp(basePetUri, theme, themeKind, petColor, petSize, petType) {
                         .join('\n'),
                 });
                 break;
+            case 'roll-call':
+                var pets = exports.allPets.pets();
+                // go through every single
+                // pet and then print out their name
+                pets.forEach((pet) => {
+                    vscode.postMessage({
+                        command: 'info',
+                        text: `Type ${pet.type}, Name ${pet.pet.name()}, Color ${pet.color} Assigned for Duty!`,
```

Comment:
`${pet.pet.emoji()} ${pet.pet.name()} (${pet.color}: ${pet.pet.hello()}`

Add a `.hello()` method to the pet interface so each pet can have a custom hello message.